acceptance/openstack/blockstorage/v3/snapshots_test.go

```diff
@@ -26,6 +27,21 @@ func TestSnapshots(t *testing.T) {
 	th.AssertNoErr(t, err)
 	defer DeleteSnapshot(t, client, snapshot1)
 
+	// Update snapshot
+	updatedSnapshotName := "ACPTTEST-002"
+	updatedSnapshotDescription := "ACPTTEST-002"
```

Comment:
nit: It would be better to use values that are randomly generated instead of hard-coded. You can randomly generate a name and description using this method: https://github.com/gophercloud/gophercloud/blob/master/acceptance/openstack/blockstorage/v3/blockstorage.go#L20-L21